src/components/OfferSwitch/OfferSwitch.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import styles from './OfferSwitch.module.scss';
+
+import type { Offer } from '#types/checkout';
+import Checkbox from '#components/Checkbox/Checkbox';
+import { formatLocalizedDate, formatPrice } from '#src/utils/formatting';
+import { useAccountStore } from '#src/stores/AccountStore';
+
+interface OfferSwitchProps {
```

Comment:
All other components are using `type`. Is there a good reason we should be using `interface` for prop definitions?

Reply:
oversight, will fix